GooglePlayInstant/LoadingScreen/LoadingBarGenerator.cs

```diff
@@ -22,20 +22,23 @@ namespace GooglePlayInstant.LoadingScreen
     /// </summary>
     public static class LoadingBarGenerator
     {
-        // Loading bar size as a proportion of the screen size. Adjust if needed.
-        private const float LoadingBarWidthProportion = 0.7f;
-        private const float LoadingBarHeightProportion = 0.02f;
+        // Loading bar size.
+        private const float LoadingBarWidth = 672;
+        private const float LoadingBarHeight = 54;
 
         // Loading bar placement as a proportion of the screen size relative to the bottom left corner. Adjust if needed.
         private const float LoadingBarPositionX = 0.5f;
-        private const float LoadingBarPositionY = 0.3f;
+        private const float LoadingBarPositionY = 0.3984375f;
 
-        // Names for the gameobject components
+        // Names for the gameobject components.
         private const string RootName = "Loading Bar";
         private const string OutlineName = "Outline";
         private const string BackgroundName = "Background";
         private const string FillName = "Fill";
         private const string ProgressName = "Progress";
+        
+        // Color of the inner fill and outline of the loading bar.
+        private static readonly Color DarkGrey = new Color32(74,74,74, 255);

```

Comment:
Nit: inconsistent spaces - run "Reformat Code"?

Reply:
Resolved, thanks for catching that.